shared_model/interfaces/base/signable.hpp

```diff
@@ -41,11 +42,10 @@ namespace shared_model {
 
       /**
        * Attach signature to object
-       * @param signature - signature object for insertion
        * @return true, if signature was added
        */
-      virtual bool addSignature(const crypto::Signed &signed_blob,
-                                const crypto::PublicKey &public_key) = 0;
+      virtual bool addSignature(types::SignedHexStringView signed_blob,
+                                types::PublicKeyHexStringView public_key) = 0;
```

Comment:
I do not think that it is a performance issue - string view consists of `const char*` and `size_t`. Copies are cheap, and there is no need to provide two interfaces, one for `&&` and another one for `const&`.

Reply:
The method is virtual, it can not be inlined and optimized. So the major penalty will be not in members copiing, but in stack frame creation and destruction while costructor call. The reference transfer will remove object creation and ctor call.